editor/export/editor_export_plugin.cpp

```diff
@@ -132,6 +132,25 @@ Variant EditorExportPlugin::get_option(const StringName &p_name) const {
 	return export_preset->get(p_name);
 }
 
+String EditorExportPlugin::_has_valid_export_configuration(const Ref<EditorExportPlatform> &p_export_platform, const Ref<EditorExportPreset> &p_preset) {
+	String warning;
+	if (!_supports_platform(p_export_platform)) {
+		return warning;
+	}
```

Comment:
I think this should result in a warning, not in an early return. Unsupported platform does mean that configuration passed to this method is invalid. The caller can safely check for this cases itself (and you already do that, actually).